### PR TITLE
chore: updated runTests.ps1 file of bigquery Quickstart

### PR DESCRIPTION
**Pull request by hemanshv**
updated quick start to use the Google_Cloud_Project env variable directly.
made the corresponding changes in  runTest.ps1 as well. 
The test is running fine locally.

### CODE REVIEW

**Review by Rishabh-V — COMMENTED** [review]
I am not sure if there is any environment in which the test for this samples run, but if there is then without the new `GOOGLE_CLOUD_PROJECT` environment variable the tests would fail. This may be fine if we work and get this variable set. However, it does introduce inconsistencies in ProjectID environment variable used here and in other samples.

We have removed the `BackupAndEdit` command. However, one way could have been to use it to replace the environment variable with `GOOGLE_PROJECT_ID`. That way, we would have made changes needed for the BQ team and also kept our tests running with the same environment variable. Please discuss with Amanda and get an agreement. Thanks.

**Comment by hemanshv** [comment]
Hi @amanda-tarafa ,
Please share your thoughts on the same.

Thanks,

**Comment by amanda-tarafa** [comment]
Why is the environment variable change necessary? As @Rishabh-V mentioned, the environment variable we use through samples is `GOOGLE_PROJECT_ID`. Why should we add a new environment variable?
Let's discuss internally please.

**Comment by amanda-tarafa** [comment]
Now that we are using an env variable, removing the replacement of the placeholder is a good change. It would have been better in #2117, but that's fine.

**Comment by hemanshv** [comment]
Hi @Rishabh-V and @amanda-tarafa ,
I've udpated the PR. Please have a look and provide your approval.

Thanks,

File: bigquery/api/SimpleApp/runTests.ps1
Comment (Rishabh-V) [inline bigquery/api/SimpleApp/runTests.ps1:16]: nit: We have removed the newline from the end of the file.

**Review by Rishabh-V — COMMENTED** [review]
A nit but this looks fine to me. Please request review from Amanda after addressing/responding to it. Thanks.

File: bigquery/api/SimpleApp/runTests.ps1
Comment (hemanshv) [reply bigquery/api/SimpleApp/runTests.ps1]: Done

**Review by amanda-tarafa — APPROVED** [review]
Please rebase all changes into a single commit before merging. Thanks.

**Review by Rishabh-V — APPROVED** [review]

**Pull request merged**
